Add custom headers to JDBC driver (#16596)
prestodb/presto: presto-docs/src/main/sphinx/installation/jdbc.rst

``extraCredentials``              Extra credentials for connecting to external services. The
                                  extraCredentials is a list of key-value pairs. Example:
                                  ``foo:bar;abc:xyz`` will create credentials ``abc=xyz`` and ``foo=bar``
``customHeaders``                 Custom headers to inject through JDBC driver. The
                                  customHeaders is a list of key-value pairs. Example:

>                                   ``testHeaderKey:testHeaderValue`` will inject the header ``testHeaderKey``
>                                   with value ``testHeaderValue``. Need to encode key-value fields individually
>                                   if there is ``:`` in the fields.

Reviewer: ```suggestion
                                  ``testHeaderKey:testHeaderValue`` will inject the header ``testHeaderKey``
                                  with value ``testHeaderValue``. Values should be percent encoded.
```

Author: ✅
